src/css/blocks/stars.css

```diff
@@ -0,0 +1,34 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  margin-bottom: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
```suggestion
  background-position: center;
  background-image: url(../../images/star.svg);
```

Reply:
> 
I didn't understand this sugestion. I tried to remove this propriety but the image was moved to the top left of the 16px square.